email_backend/templates/feedback.json

```diff
@@ -39,7 +39,7 @@
       "form_type": "radio",
       "form_label": {
         "en": "To what extent did you find GC InfoBase easy to navigate? (1=not at all useful and 5=very useful)",
-        "fr": "Dans quelle mesure la navigation est-elle facile dans l’InfoBase du GC?"
+        "fr": "Dans quelle mesure la navigation est-elle facile dans l’InfoBase du GC? (1=difficile et 5=très facile)"
```

Comment:
I think this was mixed with the other french 1-5 scale

Reply:
Woops